website/content/en/status/report-2023-07-2023-09/bsd-cafe.adoc

```diff
@@ -1,38 +1,41 @@
 Just like any bar, our discussions can encompass a wide array of topics, all while maintaining a spirit of mutual respect.
 
-BSD.cafe is poised to become a hub for a variety of tools and services, all powered by *BSD.
+BSD Cafe is poised to become a hub for a variety of tools and services, all powered by *BSD.
 
-Our initial offering is a brand-new link:https://joinmastodon.org/[Mastodon (open-source
-microblogging software and service)] instance, serving as a gateway to the Fediverse (a federation of microblogging services communicating via link:https://www.w3.org/TR/activitypub[the ActivityPub protocol]).
-Registration is now open, and the server will operate under clearly defined guidelines that promote positive conduct and unequivocally prohibit any form of hate.
-Inclusiveness, respect, and constructive dialogue stand as the cornerstones of this new instance.
+Our initial offering is a brand-new instance of link:https://joinmastodon.org/[Mastodon] (open-source
+microblogging software and service), serving as a gateway to the fediverse -- a federation of services, many of which use the link:https://www.w3.org/TR/activitypub[ActivityPub] decentralised social networking protocol.
+Registration is now open.
+The server operates under clearly defined guidelines that promote positive conduct and unequivocally prohibit any form of hate.
+Inclusiveness, respect, and constructive dialogue stand as the cornerstones of this instance.
 
 Our primary server is currently hosted in Finland on a small VM, running on FreeBSD.
-Our services are partitioned into VNET jails, interconnected within a local area network through a dedicated bridge.
-Additionally, we have implemented a VPN system and have the flexibility to migrate individual jails to more robust machines.
+Services are partitioned into VNET jails, interconnected within a local area network through a dedicated bridge.
+Additionally, we implement a VPN system and have the flexibility to migrate individual jails to more robust machines.
 
-For multimedia data and cache hosting, we have employed a separate physical server (also FreeBSD-based, within a jail), fronted by Cloudflare.
+For multimedia data and cache hosting, we employ a separate physical server (also FreeBSD-based, within a jail), fronted by Cloudflare.
 The goal here is to cache and geodistribute data, effectively reducing network congestion on the main VPS.
 
-Our reverse proxy (frontend), mail server, media server, and the instance itself are all accessible via ipv6.
+Our reverse proxy (frontend), mail server, media server, and the instance itself are all accessible via IPv6.
 
-From the moment of its inception, this instance has remained devoid of preloaded content.
-Our intention is for it to grow organically based on the interests and followers of its users.
+At its inception, this Mastodon instance was devoid of preloaded content.
+Our intention is for it to grow organically, based on the interests and followers of its users.
```

Comment:
This does not yet make sense to me: 

> … grow …, based on the … followers of its users.

Reply:
Like this, maybe? 

> … grow organically, based on the interests of its users and followers.

Reply:
I am not familiar with Mastodon, but I believe it is a social network where users have followers. I think what is intended here is that the instance should grow using two elements:
- the interests of the users, which will increase the amount of topics discussed;
- the followers of the users, who will increase the amount of people discussing the topics.

Reply:
Succinctly: 

* Mastodon is broadly comparable to X (formerly Twitter), but not restricted to a single domain (e.g. `twitter.com`)
* in the screenshot below (a selection from my _BSD.cafe Mastodon Portal_ preferences), only one of five accounts is in the `mastodon.bsd.cafe` domain.

![image](https://github.com/freebsd/freebsd-doc/assets/192271/db5ffa2a-2d9c-4118-aaec-12f5ad9ac2cd)

HTH

Reply:
https://github.com/freebsd/freebsd-doc/pull/255#issuecomment-1738536885 helps to explain; to resolve this conversation.